src/index.ts

```diff
@@ -213,6 +247,18 @@ const main = (vortexExt: VortexExtensionContext): boolean => {
     ),
   );
 
+  const availableStartHooks = prepStartHooks(vortexExt, fullFeatureSetAvailablePostStartup);
+
+  pipe(
+    availableStartHooks,
+    mapWithIndex((i: number, { hookId, transformRunParams }) =>
+      forEffect(() => { vortexExt.registerStartHook(40 + i, hookId, transformRunParams); })),
```

Comment:
Why 40? Could it be any number, and that is just the first one you thought of? Not 69? Am I a child for that being the second number I think of when I need a number?

Reply:
meh. It's probably fine as is.